### PR TITLE
Add some themes

### PR DESCRIPTION
**Pull request by hyphen-o**
To make this app more colorful I implemented adding "yellow" and "invert" themes. Invert means white for the light theme and black for the dark theme.
This is my first implementation with Swift,so please close this PR if this does not work as you expect😅

Thanks for a great app!

### CODE REVIEW

File: GitHubContributionsWidget/GitHubContributionsGraphWidget.swift [hunk 1]
```diff
@@ -19,6 +19,6 @@ struct GitHubContributionsGraphWidget: Widget {
         .configurationDisplayName("widget-graph-display-name")
         .description("widget-graph-description")
         .supportedFamilies([.systemSmall, .systemMedium])
-        .contentMarginsDisabled()
+       .contentMarginsDisabled()
```
Comment (AnderGoig) [inline GitHubContributionsWidget/GitHubContributionsGraphWidget.swift:22]: can we revert this change?

File: GitHubContributionsWidget/Extensions/ThemeExtensions.swift [hunk 2]
```diff
@@ -26,6 +26,8 @@ extension Theme {
         case .blue: return .blueLevel1
         case .red: return .redLevel1
         case .purple: return .purpleLevel1
+        case .yellow: return .yellowLevel1
+        case .invert: return .invertLevel1
```
Comment (AnderGoig) [inline GitHubContributionsWidget/Extensions/ThemeExtensions.swift:30]: what do you think about renaming the "Invert" them to "Inverted"?

File: GitHubContributionsWidget/es.lproj/GitHubContributionsWidget.strings [hunk 3]
```diff
@@ -24,6 +28,10 @@
 
 "O7wabU" = "Morado";
 
+"ub2khZ" = "Amarillo";
+
+"ywLI12" = "Inversión";
```
Comment (AnderGoig) [inline GitHubContributionsWidget/es.lproj/GitHubContributionsWidget.strings:33]: I would maybe translate it to "Invertido" in spanish 🤔

**Review by AnderGoig — CHANGES_REQUESTED** [review]
Hey @hyphen-o, awesome contribution! Thank you so much and sorry for the delay reviewing this.

I love the inverted theme and the yellow one looks amazing on dark mode. 

However, when I switch to light mode, the yellow one looks a bit too light. Perhaps a bit more yellowish color, or darker yellow would work? what do you think? 🤔

File: GitHubContributionsWidget/Extensions/ThemeExtensions.swift (hunk 2 above)
Comment (hyphen-o) [reply GitHubContributionsWidget/Extensions/ThemeExtensions.swift]: fixed at https://github.com/AnderGoig/github-contributions-ios/pull/20/commits/64cd790b0406ce2c37c65d610546c3d0ce1ae2ce

File: GitHubContributionsWidget/es.lproj/GitHubContributionsWidget.strings (hunk 3 above)
Comment (hyphen-o) [reply GitHubContributionsWidget/es.lproj/GitHubContributionsWidget.strings]: fixed at https://github.com/AnderGoig/github-contributions-ios/pull/20/commits/64cd790b0406ce2c37c65d610546c3d0ce1ae2ce

**Comment by hyphen-o** [comment]
Thanks for reviewing!
Fixed the yellow color theme.
How do you like it?
スクリーンショット 2024-02-13 11 45 54

**Review by AnderGoig — APPROVED** [review]
> Thanks for reviewing! Fixed the yellow color theme. How do you like it?

much better! thanks @hyphen-o
